documentation/modules/creating-plan-wizard-290-cnv.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="creating-plan-wizard-290-cnv_{context}"]
+
+= Creating an {virt} migration plan by using the MTV wizard
+
+You can migrate {virt} virtual machines (VMs) by using the {project-full} plan creation wizard.
+
+The wizard is designed to lead you step-by-step in creating a migration plan.
+
+[WARNING]
+====
+Do not include virtual machines with guest-initiated storage connections, such as Internet Small Computer Systems Interface (iSCSI) connections or Network File System (NFS) mounts. These require either additional planning before migration or reconfiguration after migration.
+
+This prevents concurrent disk access to the storage the guest points to.
+====
+
+include::snip_plan-limits.adoc[]
+
+[IMPORTANT]
+====
+When you click *Create plan* on the *Review and create* page of the wizard, {project-first} validates your plan. If everything is OK, the *Plan details* page for your plan opens. This page contains settings that do not appear in the wizard, but are important. Be sure to read and follow the instructions for this page carefully, even though it is outside the plan creation wizard. The page can be opened later, any time before you run the plan, so you can come back to it if needed.
+====
+
+.Prerequisites
+
+* Have an {virt} source provider and a {virt} destination provider. For more information, see xref:adding-source-provider_cnv[Adding an {virt} source provider] or xref:adding-source-provider_dest_cnv[Adding {a-virt} destination provider].
+* If you plan to create a *Network map* or a *Storage map* that will be used by more than one migration plan, create it in the *Network maps* or *Storage maps* page of the UI before you create a migration plan that uses that map.
+
+.Procedure
+
+. In the {ocp} web console, click *Migration for Virtualization* > *Migration plans*.
+. Click *Create plan*.
++
+The *Create migration plan* wizard opens.
++
+// General page
+. On the *General* page, specify the following fields:
+
+* *Plan name*: Enter a name.
+* *Plan project*: Select from the list.
+* *Source provider*: Select from the list.
+* *Target provider*: Select from the list.
+* *Target project*: Select from the list.
+
+. Click *Next*.
++
+// Virtual machines page
+. On the *Virtual machines* page, select the virtual machines you want to migrate and click *Next*.
++
+// Network map page
+. On the *Network map* page, choose one of the following options:
+
+* *Use an existing network map*: Select an existing network map from the list.
++
+These are network maps available for all plans, and therefore, they are _ownerless_ in terms of the system. If you select this option and choose a map, a copy of that map is attached to your plan, and your plan is the _owner_ of that copy. Any changes you make to your copy do not affect the original plan or any copies that other users have.
```

Comment:
`These are network maps available for all plans` is not accurate, we do need to consider that the plan source and target providers should be the same like the source and target providers in the mappings.

Reply:
@avivtur When a user clicks "Use an existing network map," does the UI display **only** maps with the same source and target providers or should I advise users to check this before going further? 
@mnecas @mmenestr any comments?

Reply:
@RichardHoch no it only filters the un-owned ones so we should warn the users about it, we should probably fix that in the next z-stream,
@mnecas @mmenestr I can't understand why we limit this to un-owned storage/network map, the current flow will create a copy of the mapping, is there a reason we don't create a copy of owned resources? IMO the limitation should be on the providers not the owned/un-owned

Reply:
@avivtur Thanks for the clarification -- I'll add a warning (as a "Note" to check this) and send the PR back to you.